client/src/components/Warning/Warning.tsx

```diff
@@ -0,0 +1,28 @@
+import { Row } from 'antd';
+import Image from 'next/image';
+import { PageLayout } from 'components/PageLayout';
+
+const defaultMessage = 'Something went wrong, please try reloading the page later';
+const defaultImageName = 'Image error';
+
+type Props = {
+  loading: boolean;
+  githubId: string;
+  imagePath: string;
+  nameImage: string;
+  textMessage: string | JSX.Element;
+};
+
+export function Warning(props: Props) {
+  const { loading = false, githubId, imagePath, nameImage = defaultImageName, textMessage = defaultMessage } = props;
```

Comment:
Please consider to choose 1 option: optional props & default values || mandatory props & no default values :)